cs3220_lab4_fixed/src/cs3220_lab4/FoodItemEntry.java

```diff
@@ -0,0 +1,43 @@
+package cs3220_lab4;
+
+public class FoodItemEntry {
+	public final int id;
+	public final String name;
+	public final String description;
+	public final String url;
+	public final Double price;
+	public final String date;
```

Comment:
e.g. if you want to sort by date on the food items, you can sort with Date easier than just sorting by string.